include/seastar/core/shared_future.hh

```diff
@@ -239,6 +239,10 @@ private:
         bool failed() const noexcept {
             return _original_future.failed();
         }
+
```

Comment:
could add a comment to explain that this helper is only provided for testing.

Reply:
Done